config/routes.rb

```diff
@@ -416,6 +416,12 @@
           put 'test_connection'
         end
         resources :images, :except => [:show]
+        resources :key_pairs, :only => [:index, :create] do
+          member do
+            get 'download'
+            delete 'remove'
```

Comment:
sorry for catching this pretty late, but this should simply default to rest destroy action